### PR TITLE
fix: Update renovate file

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Description

Update renovate file to extend the renovate file from metadata-injection repo

## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature / enhancement (non-breaking change which adds functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] Add changelog entry following the [contributing guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

### CODE REVIEW

File: .github/renovate.json5 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": [
-    "github>newrelic/coreint-automation:renovate-base.json5"
+    "github>newrelic/k8s-metadata-injection:renovate-base.json5"
   ],
   ignoreDeps: ["ingress-nginx/kube-webhook-certgen"],
```
Comment (juanjjaramillo) [inline .github/renovate.json5:5]: If you are including this line on the shared config, it is not needed here anymore

File: .github/renovate.json5 [hunk 2]
```diff
@@ -12,19 +12,5 @@
       "groupName": "Minikube version (batch update)",
       "matchDepNames": ["manusa/actions-setup-minikube", "kubernetes/minikube"],
     },
-  ],
```
Comment (juanjjaramillo) [inline .github/renovate.json5:15]: This `packageRules` should be deleted and moved to the shared config file following my comment here: https://github.com/newrelic/k8s-metadata-injection/pull/454#discussion_r1388426955

**Review by juanjjaramillo — CHANGES_REQUESTED** [review]

File: .github/renovate.json5 (hunk 1 above)
Comment (svetlanabrennan) [reply .github/renovate.json5:5]: Ah that's right. I'll remove it.

File: .github/renovate.json5 (hunk 2 above)
Comment (svetlanabrennan) [reply .github/renovate.json5:15]: Done

**Review by juanjjaramillo — APPROVED** [review]

**Pull request merged**